web_src/js/features/comp/ComboMarkdownEditor.js

```diff
@@ -114,13 +115,30 @@ class ComboMarkdownEditor {
     }
   }
 
-  setupDropzone() {
+  async setupDropzone() {
     const dropzoneParentContainer = this.container.getAttribute('data-dropzone-parent-container');
     if (dropzoneParentContainer) {
       this.dropzone = this.container.closest(this.container.getAttribute('data-dropzone-parent-container'))?.querySelector('.dropzone');
+      if (this.dropzone) this.attachedDropzoneInst = await initDropzone(this.dropzone);
     }
   }
 
+  dropzoneGetFiles() {
+    if (!this.dropzone) return null;
```

Comment:
```suggestion
    if (!this.dropzone) return [];
```

Combine with change below.

Reply:
No...................

Non-existing is non-existing, `[]` is different from null

Reply:
Empty array means "no files" and if this were typescript it would be easier to type with `Array` return value instead of `Array | undefined`.

Reply:
No, non-existing is non-existing, null is null.

I am sure my approach is right because I have home-made a lot of frameworks.

Reply:
It's more of a matter of taste really. With typescript I come to prefer simpler function signatures over returning values such as `null`, but I won't block on it.